lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py

```diff
@@ -26,9 +26,20 @@
     name:
         description:
             - Limit results to a specific virtual machine scale set
+        required: false
```

Comment:
if name is not required, what's the unique id to get a vmss?

Reply:
it will return list of vmss in the resource group

Reply:
required: false is the default and isn't needed